### DISCUSSION TITLE
ASA-BIOP Covariate Adjustment SWG - Journal Club: Discussion on the FDA Guidance for Covariate Adjustment

### DISCUSSION THREAD

**Discussion by giscus** [quarto-dev/quarto-web#1065] [General]
# ASA-BIOP Covariate Adjustment SWG - Journal Club: Discussion on the FDA Guidance for Covariate Adjustment

March 2024 Journal Club

http://localhost:7129/posts/journalclub1.html

**Comment by hongfei-li** [comment]
test2
